jupyter-js-widgets/src/utils.js

```diff
@@ -66,14 +66,18 @@ function loadClass(class_name, module_name, registry) {
 
             // If the module is jupyter-js-widgets, we can just self import.
             var modulePromise;
-            if (module_name === 'jupyter-js-widgets') {
-                modulePromise = Promise.resolve(require('../'));
-            } else {
+            if (typeof requirejs !== 'undefined') {
                 modulePromise = new Promise(function(innerResolve, innerReject) {
-                    window.require([module_name], function(module) {
+                    requirejs([module_name], function(module) {
```

Comment:
window.require is probably better because it does not get picked up by webpack *and* you don't need to define requirejs.